config/crd/bases/atlas.mongodb.com_atlasclusters.yaml

```diff
@@ -344,8 +344,36 @@ spec:
                   - type
                   type: object
                 type: array
+              connectionStrings:
+                description: ConnectionStrings configuration for applications use
+                  to connect to this cluster
+                properties:
+                  awsPrivateLink:
+                    additionalProperties:
+                      type: string
+                    type: object
+                  awsPrivateLinkSrv:
+                    additionalProperties:
+                      type: string
+                    type: object
+                  private:
+                    type: string
+                  privateSrv:
+                    type: string
+                  standard:
+                    type: string
+                  standardSrv:
+                    type: string
+                type: object
+              mongoDBVersion:
+                type: string
+              mongoURIUpdated:
+                type: string
+              stateName:
+                type: string
             required:
             - conditions
+            - connectionStrings
```

Comment:
not required

Reply:
Is there any info on how status fields are validated? I.e. since status doesn't contain user input, do we still need to add full validation to it?

Reply:
good question, I think in general we don't need validation for the status as user cannot/should not change it. So maybe we need to remove `conditions` as well